partcad/src/partcad/assembly_factory_assy.py

```
@@ -809,7 +831,23 @@ async def handle_node(self, assembly, node):
+                source_frame = None if source_port is None else source_port.location
+                mated_frame = location if source_frame is None else location * source_frame
+                connect_how.resolve(
+                    item,
+                    target_part,
+                    source_frame=source_frame,
+                    mated_frame=mated_frame,
+                    source_interface=source_iface_obj,
+                    target_interface=target_iface_obj,
+                )
+
         if not item is None:
```

Ruff (0.16.1)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)